docs/apm-distributed-tracing.asciidoc

```diff
@@ -42,24 +42,27 @@ and continues to propagate the trace.
 In this example, Elastic's Ruby agent communicates with Elastic's Java agent.
 Both support the `traceparent` header, and trace data is successfully propagated.
 
+// lint ignore traceparent
 image::./images/dt-trace-ex1.png[How traceparent propagation works]
 
 In this example, Elastic's Ruby agent communicates with OpenTelemetry's Java agent.
 Both support the `traceparent` header, and trace data is successfully propagated.
 
+// lint ignore traceparent
 image::./images/dt-trace-ex2.png[How traceparent propagation works]
 
 In this example, the trace meets a piece of middleware that doesn't propagate the `traceparent` header.
 The distributed trace ends and any further communication will result in a new trace.
 
+// lint ignore traceparent
 image::./images/dt-trace-ex3.png[How traceparent propagation works]
```

Comment:
Instead of lint ignoring these, can we just change the alt text to be:

> How `traceparent` propagation works

Reply:
I tried that, but it doesn't work. I think it's because this becomes the `alt` in the `img` in HTML, which has to be a string -- it doesn't account for `code` formatting.

Reply:
Ahh, I see. Cool beans.